### DISCUSSION TITLE
select vs poll and file descriptor limits

### DISCUSSION THREAD

**Discussion by jmckitrick** [daviderestivo/homebrew-emacs-head#229] [General]
I was intrigued by the `--with-poll` feature which would remove the limitation of 1024 file descriptors under macOS, but I recently noticed the formula already bumps up the limit when building emacs. After briefly researching 'select vs poll', it seems the biggest benefit is removing that limitation. What other advantages does 'poll' offer on macOS?

**Comment by daviderestivo** [comment]
Ciao @jmckitrick, 

to be honest I never had the time to investigate it in detail. I guess it's already in master... am I right?
did you already build emacs with this option and use it?

Regards,
Davide

**Comment by jmckitrick** [comment]
Hi @daviderestivo I thought the 'poll' feature would be useful if it allowed more open files and connections, but I found an issue using CIDER with it. The author says he no longer has time to work on the feature and does not know when it will resume. When I saw the formula increases the number of file descriptors anyway, I wondered if there were any important differences between the 2 approaches.
